src/main/app/controller/PaymentController.ts

```diff
@@ -0,0 +1,84 @@
+import config from 'config';
+import dayjs from 'dayjs';
+import { Response } from 'express';
+
+import { APPLICATION_SUBMITTED, HOME_URL, PAYMENT_CALLBACK_URL } from '../../steps/urls';
+import { CITIZEN_ADD_PAYMENT, CITIZEN_SUBMIT, PaymentStatus, State } from '../case/definition';
+import { Payment, PaymentClient } from '../payment/PaymentClient';
+import { PaymentModel } from '../payment/PaymentModel';
+
+import { AppRequest } from './AppRequest';
+
+export class PaymentController {
```

Comment:
move this to the step as a default export and move the callback controller to a separate step